modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -1651,7 +1651,7 @@ label:dynamic[Dynamic]
 [frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
 |===
 |Description
-a|Database format. This is the format that will be used for new databases. Valid values are `standard`, `aligned`, or `high_limit`.The `aligned` format is essentially the `standard` format with some minimal padding at the end of pages such that a single record will never cross a page boundary. The `high_limit` format is available for Enterprise Edition only. It is required if you have a graph that is larger than 34 billion nodes, 34 billion relationships, or 68 billion properties.
+a|Database format. This is the format that will be used for new databases. xref:tools/neo4j-admin/neo4j-admin-store-info.adoc#_store_formats_and_entity_limits[Valid values] are `standard`, `aligned`, or `high_limit`. The `aligned` format is essentially the `standard` format with some minimal padding at the end of pages such that a single record will never cross a page boundary. The `high_limit` format is available for Enterprise Edition only. It is required if you have a graph that is larger than 34 billion nodes, 34 billion relationships, or 68 billion properties.
```

Comment:
Other configs have links in the description, for example this one https://neo4j.com/docs/operations-manual/5/configuration/configuration-settings/#config_dbms.databases.seed_from_uri_providers. That's why I thought it would be fine?

Reply:
If we want to keep it, we need to add it to the code base as well.